### PR TITLE
MOTECH-2839: Cherry-pick- Added Cohort Query Member Task Trigger

### PR DESCRIPTION
**Pull request by afijal**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

File: openmrs/src/main/java/org/motechproject/openmrs/tasks/constants/DisplayNames.java [hunk 1]
```diff
@@ -55,13 +55,13 @@
     public static final String PROGRAM_ENROLLMENT_ATTRIBUTES = "openMRS.programEnrollment.attributes";
 
     //CohortQuery Report action
```
Comment (kmadej) [inline openmrs/src/main/java/org/motechproject/openmrs/tasks/constants/DisplayNames.java]: just CohortQuery

**Comment by kmadej** [comment]
test this please

**Comment by kmadej** [comment]
test this please

**Comment by kmadej** [comment]
test this please

**Comment by kmadej** [comment]
test this please

**Pull request merged**